packages/manager/apps/dedicated/client/app/ip/components/list/block/move/ip-ip-block-move.controller.js

```diff
@@ -187,5 +271,21 @@ export default /* @ngInject */ (
     return $scope.loading.save;
   };
 
+  $scope.watingTask = (taskId, serviceName, callback) => {
+    OvhApiVrack.v6()
+      .task({
+        serviceName,
+        taskId,
+      })
+      .$promise.then(() => {
+        $timeout(() => {
+          $scope.watingTask(taskId, serviceName, callback);
+        }, POLLING_INTERVAL);
```

Comment:
**nitpick:** `$interval` is the right method to use for polling. Also, there exists a PollingService on Manager, you can check if it can be used.